site/docs/getting-started/tutorials/continuous-materialized-view.md

```diff
@@ -13,9 +16,9 @@ a materialized view that updates continuously based on a real-time data feed.
 
 * An Estuary Flow account. If you don't have one, visit the [Flow web app](https://dashboard.estuary.dev) to register for free.
 
-* The [flowctl CLI installed](../installation.mdx#get-started-with-the-flow-cli) (for the optional section).
+* A GitLab, GitHub, or BitBucket account. You'll use this to log into [GitPod](https://www.gitpod.io/), the cloud development environment integrated with Flow.
```

Comment:
My 2 cents is that we should still at least mention that installing flowctl locally is an option. Gitpod is probably still easiest for most people, but it feels pretty weird for that to be the _only_ option.

If you did that, then we'd probably also need another note somewhere near the derivation YAML that says something like "if you're using flowctl locally, just paste this into a flow.yaml file and change 'yourprefix' to your tenant name".

I'm not super strongly attached to this, but it's definitely what I'd prefer if I was following the tutorial myself.